src/Components/MyProfile.jsx

```diff
@@ -12,22 +180,38 @@ class MyProfile extends Component {
               My profile
             </Header>
             <br></br>
-            <Container className="align-center">
-              <p >
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-              </p>
-              <Divider hidden />
+            <Grid className="align-center">
+              <Grid.Column>
               <p>
                 Please help us make Being Stockholm better and answer six questions about yourself.
```

Comment:
Could we change this to 30 seconds or one minute instead of "six questions", that may change in time :)

Reply:
done